Makefile

```diff
@@ -183,6 +186,22 @@ ci-rustdoc:
 audit:
 	@for f in `./tools/list_lock.sh`; do echo "$$(tput bold)Auditing $$f"; (cd "$$f" && cargo audit || exit 1); done
 
+.PHONY: emulation-setup
+emulation-setup:
+	@#Use the latest QEMU as it has OpenTitan support
+	@if [[ ! -d tools/qemu || ! -f tools/qemu/VERSION ]]; then \
+		rm -rf tools/qemu; \
+		cd tools; git clone https://github.com/qemu/qemu.git; \
+		cd qemu; ./configure --target-list=riscv32-softmmu; \
+	fi
+	@$(MAKE) -C "tools/qemu" > /dev/null
+
+.PHONY: emulation-check
+emulation-check: emulation-setup
+	@$(MAKE) -C "boards/hifive1"
+	@PATH="${PATH}:../../tools/qemu/riscv32-softmmu/" timeout --foreground 5s $(MAKE) qemu -C "boards/hifive1" | tee log > /dev/null
```

Comment:
Can we somehow make this more self-contained so that it doesn't just drop a `log` file, maybe follow the technique described here: https://unix.stackexchange.com/a/181938/12313 ?